src/Homepage/ResultBlock.js

```diff
@@ -81,6 +82,11 @@ function ResultBlock(props) {
               textToHighlight={result.name}
             />
           </h5>
+          <div className="mt-n1">
+            <small className="text-muted d-inline-block mt-2">
+              <strong>Within {distanceKM} km</strong>
```

Comment:
can we use translations for this string, so that it can be localized in the different languages?

Reply:
Sure, I'll work on it